fuse/server_darwin.go

```diff
@@ -30,3 +32,35 @@ func (ms *Server) systemWrite(req *request, header []byte) Status {
 	}
 	return ToStatus(err)
 }
+
+// for a stream connection we need to have a single reader
+var readLock sync.Mutex
+
+func (ms *Server) systemRead(dest []byte) (int, error) {
+	if osxFuse {
+		return syscall.Read(ms.mountFd, dest)
```

Comment:
Look like you lost handleEINTR in this branch

Reply:
Correct, I'll re-submit.